### PR TITLE
90dns and missingconfig

### PR DESCRIPTION
**Pull request by JeffVi**
Changed both 90dns and missingconfig switch assistance commands

### 90DNS
Removed the IP addresses to mention the preferred method (dns.mitm)

### missingconfig
Made it clear that you have to replace your hekate_ipl.ini by the one provided

### CODE REVIEW

**Comment by JeffVi** [comment]
Reverted changes to .90dns to add a new command instead (.dnsmitm), slightly reworded .missingconfig, and updated nx version cause why not

File: cogs/assistance-cmds/dnsmitm.switch.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+---
+title: DNS.mitm
+help-desc: Applying AMS DNS redirection
+---
+
+To ensure that your emuMMC/emuNAND does not connect to Nintendo, you should use DNS.mitm (AMS DNS redirection).
+
+[Follow these steps](https://nh-server.github.io/switch-guide/extras/blocking_updates/) to set up DNS.mitm on your emuMMC/emuNAND and redirect requests to Nintendo.
```
Comment (el-emmo) [inline cogs/assistance-cmds/dnsmitm.switch.md:8]: "...redirect requests to Nintendo."

This is unclear as we're not redirecting the requests to Nintendo. Maybe something like "...redirect requests directed to Nintendo to your SD card/to nothing instead"?

Comment (JeffVi) [reply cogs/assistance-cmds/dnsmitm.switch.md]: Used the wording of the guide

**Comment by JeffVi** [comment]
Waiting for the name change of the "blocking updates" page

**Comment by JeffVi** [comment]
Ready to be merged
[Switch guide got updated](https://github.com/nh-server/switch-guide/pull/183)

**Pull request merged**